fee_allocator/accounting/overrides.py

```diff
@@ -0,0 +1,64 @@
+from abc import ABC, abstractmethod, ABCMeta
+from decimal import Decimal
+from typing import Dict, Type, TYPE_CHECKING
+from fee_allocator.constants import POOL_OVERRIDES_URL
+import requests
+
+if TYPE_CHECKING:
+    from fee_allocator.accounting.core_pools import PoolFee
+
+overrides_data = requests.get(POOL_OVERRIDES_URL).json()
+
+
+class PoolFeeOverrideMeta(ABCMeta):
+    overrides: Dict[str, Type["PoolFeeOverride"]] = {}
+
+    def __new__(mcs, name, bases, attrs):
+        cls = super().__new__(mcs, name, bases, attrs)
+        if name != "PoolFeeOverride":
+            pool_id = attrs.get("pool_id")
+            if pool_id:
+                mcs.overrides[pool_id] = cls
+        return cls
+
+
+class PoolFeeOverride(ABC, metaclass=PoolFeeOverrideMeta):
+    POOL_ID: str = None
+    voting_pool: str = None
+    market: str = None
+
+    def __init__(self, core_pool: "PoolFee"):
+        self.core_pool = core_pool
+
+    @property
+    @abstractmethod
+    def to_aura_incentives_usd(self) -> Decimal:
+        pass
+
+    @property
+    @abstractmethod
+    def to_bal_incentives_usd(self) -> Decimal:
+        pass
+
+
+class RethWethOverride(PoolFeeOverride):
```

Comment:
I don't think I like the idea of building individual overrides as classes.  It's better to have them grouped by type, and then a single json file where they can all be easily seen in config.

Overrides are config not code, there needs to be  well documented code to handle the config, but the overrides themselves should remain in json files, and likely in another repo like multisig-ops with different change control.

Reply:
yes assuming we just want the overrides to be static, all or nothing overrides like we have now, then this isnt necessary